longestPalindrome.py

```diff
@@ -0,0 +1,36 @@
+"""
+Approach: 
+I solved this using a greedy approach with a set.
+we will iterate through the string and if there no occrance we add the char to the set
+else we know that is already in the set. we increment the count by 2 and we remove the char from the set
+if still there are ant chars in the set, then we can create a odd num of palindrome like "aba". so we increment the count by 1
+in the end we return count which gives the longest palindrome
+
+Time Complexity: O(n)
+
+Space Complexity: O(n)
```

Comment:
Please check space complexity. Since you will have constant number of characters (at max all valid ASCII chars) in set , so space would be constant